src/middlewares/user.ts

```diff
@@ -13,3 +17,14 @@ export const isNotInATeam = async (request: UserRequest, response: Response, nex
 
   return next;
```

Comment:
```suggestion
  return next();
```
c'est pas tes modifs, mais en passant ^^